bcda/models/models.go

```diff
@@ -347,6 +348,19 @@ type CCLFBeneficiary struct {
 	BlueButtonID string `gorm:"type: text"`
 }
 
+type Suppression struct {
+	gorm.Model
+	HICN                string `gorm:"type:varchar(11);not null"`
+	SourceCode          string `gorm:"type:varchar(5)"`
+	EffectiveDt         time.Time
+	PrefIndicator       string `gorm:"type:char(1)"`
+	SAMHSASourceCode    string `gorm:"type:varchar(5)"`
+	SAMHSAEffectiveDt   time.Time
+	SAMHSAPrefIndicator string `gorm:"type:char(1)"`
+	ACOCMSID            string `gorm:"type:char(5)"`
```

Comment:
I just tried a GORM migration locally, and the column names all look correct except `ACOCMSID` (ends up as `acocms_id`). So we probably need to specify the name for that one.

Reply:
ooo nice, good check too. Will update